codecov.yml

```diff
@@ -0,0 +1,4 @@
+coverage:
```

Comment:
I don't think it's the intention to turn off the coverage for the project. In general coverage should almost never go down, we also had no previous issues with this.